src/schema/BitmapSchema.ts

```diff
@@ -4,45 +4,59 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Schema } from "./Schema.js";
+import { ByteArray } from "../util/ByteArray";
+import { Schema } from "./Schema";
 
 const enum BitRangeType {
     Flag,
     Number,
     Enum,
 }
-type BitRange<T> = { type: BitRangeType, offset: number, mask: number };
-const BitRange = <T>(type: BitRangeType, offset: number, length: number) => ({ type, mask: ((1 << length) - 1) << offset, offset } as BitRange<T>);
+
+type BitRange<T> = { type: BitRangeType, offset: number, length: number };
+const BitRange = <T>(type: BitRangeType, offset: number, length: number ) => ({ type, offset, length } as BitRange<T>);
 
 /** Defines the bit position of a boolean flag. */
 export interface BitFlag extends BitRange<boolean> { type: BitRangeType.Flag }
 export const BitFlag = (offset: number) => BitRange(BitRangeType.Flag, offset, 1) as BitFlag;
 
 /** Defines the bit position and bit length of a numeric value. */
 export interface BitField extends BitRange<number> { type: BitRangeType.Number }
-export const BitField = (offset: number, length: number) => BitRange(BitRangeType.Number, offset, length) as BitField;
+export const BitField = (offset: number, length: number ) => BitRange(BitRangeType.Number, offset, length) as BitField;
```

Comment:
Is this trailing space before `)` imposed by lint / style guide?  Or just a stray character?

Reply:
Just a stray... removed!